projects/storefrontstyles/scss/components/checkout/steps/review/_review-submit.scss

```diff
@@ -14,12 +14,25 @@
     margin: 0;
     background-color: var(--cx-color-background);
 
+    &-card {
+      min-height: 180px;
+    }
+
+    &-edit-step {
+      padding: 0 20px 20px;
+
+      @include media-breakpoint-down(md) {
```

Comment:
![image](https://user-images.githubusercontent.com/8040837/63428349-a733ea00-c417-11e9-8aab-11db77f1136d.png)
I think that should be aligned to bottom

Reply:
fixed:
![image](https://user-images.githubusercontent.com/22354613/63693775-37f23780-c815-11e9-9fcf-4d8ed9b21dea.png)